src/streamlink/stream/ffmpegmux.py

```diff
@@ -91,13 +91,14 @@ def __init__(self, session, *streams, **options):
                              for stream, np in
                              zip(self.streams, self.pipes)]
 
-        ofmt = options.pop("format", "matroska")
+        ofmt = session.options.get("ffmpeg-fout") or options.pop("format", "matroska")
         outpath = options.pop("outpath", "pipe:1")
         videocodec = session.options.get("ffmpeg-video-transcode") or options.pop("vcodec", "copy")
         audiocodec = session.options.get("ffmpeg-audio-transcode") or options.pop("acodec", "copy")
         metadata = options.pop("metadata", {})
         maps = options.pop("maps", [])
         copyts = options.pop("copyts", False)
+        start_at_zero = session.options.get("ffmpeg-start-at-zero", True)
```

Comment:
`TypeError: get() takes 2 positional arguments but 3 were given`

there is no default

Reply:
Good spot.

Reply:
Perhaps we should extend `Options` to support a default, easy to do without making any breaking changes. 

```diff
diff --git a/src/streamlink/options.py b/src/streamlink/options.py
index 5a208c16..0f442d48 100644
--- a/src/streamlink/options.py
+++ b/src/streamlink/options.py
@@ -33,10 +33,11 @@ class Options(object):
     def set(self, key, value):
         self.options[_normalise_option_name(key)] = value

-    def get(self, key):
+    def get(self, key, default=None):
         key = _normalise_option_name(key)
         if key in self.options:
             return self.options[key]
+        return default

     def update(self, options):
         for key, value in options.items():
```